### PR TITLE
NAS-100260 Updated intro to include the new quota exceeded on dataset option.

### PR DESCRIPTION
**Pull request by a-stjohn**
(cherry picked from commit 174ecc61526ab0b92f7b0c954c09121a4d745093)

### CODE REVIEW

File: userguide/intro.rst [hunk 1]
```diff
@@ -96,6 +96,10 @@ fixes.
 
 These major features are new in this version:
 
+* Change if and when an alert is shown when using space on a dataset.
```
Comment (wblock) [inline userguide/intro.rst]: If and when?  "when using space"?  Maybe a capacity alert?

File: userguide/storage.rst [hunk 2]
```diff
@@ -747,11 +747,31 @@ clicking the :guilabel:`Edit Options` button in
    |                       |                 | prevent users from taking all available space.                                                              |
    |                       |                 |                                                                                                             |
    +-----------------------+-----------------+-------------------------------------------------------------------------------------------------------------+
+   | Quota warning         | integer         | Only available in :guilabel:`Advanced Mode`.                                                                |
```
Comment (wblock) [inline userguide/storage.rst]: Any table with Advanced Mode should use checkmarks in a column.

File: userguide/storage.rst [hunk 3]
```diff
@@ -747,11 +747,31 @@ clicking the :guilabel:`Edit Options` button in
...
+   | alert at, %           |                 | Issues a warning alert when the dataset quota has reached the specifed value in percent.                    |
```
Comment (wblock) [inline userguide/storage.rst]:
```suggestion
   | alert at, %           |                 | Show an alert when the dataset quota reaches the specified value in percent. 
```

File: userguide/storage.rst [hunk 4]
```diff
@@ -747,11 +747,31 @@ clicking the :guilabel:`Edit Options` button in
...
+   |                       |                 | To inherit the dataset values of the parent, leave blank. To disable this setting, enter *0*.               |
```
Comment (wblock) [inline userguide/storage.rst]:
```suggestion
   |                       |                 | Leave blank to inherit parent dataset value, or enter *0* to disable.
```

File: userguide/storage.rst [hunk 5]
```diff
@@ -747,11 +747,31 @@ clicking the :guilabel:`Edit Options` button in
...
+   |                       |                 |                                                                                                             |
+   +-----------------------+-----------------+-------------------------------------------------------------------------------------------------------------+
+   | Quota critical        | integer         | Only available in :guilabel:`Advanced Mode`.                                                                |
```
Comment (wblock) [inline userguide/storage.rst]: Checkbox for column.

File: userguide/storage.rst [hunk 6]
```diff
@@ -747,11 +747,31 @@ clicking the :guilabel:`Edit Options` button in
...
+   | alert at, %           |                 | Issues a critical alert when the dataset quota has reached the specified value in percent.                  |
```
Comment (wblock) [inline userguide/storage.rst]: As above.

File: userguide/storage.rst [hunk 7]
```diff
@@ -747,11 +747,31 @@ clicking the :guilabel:`Edit Options` button in
...
+   |                       |                 | To inherit the dataset values of the parent, leave blank. TO disable this setting, enter *0*.               |
```
Comment (wblock) [inline userguide/storage.rst]: As above.

File: userguide/storage.rst [hunk 8]
```diff
@@ -747,11 +747,31 @@ clicking the :guilabel:`Edit Options` button in
...
+   +-----------------------+-----------------+-------------------------------------------------------------------------------------------------------------+
    | Quota for this        | integer         | Only available in :guilabel:`Advanced Mode`.                                                                |
    | dataset and all       |                 | A specified value applies to both this dataset and any child datasets.                                      |
    | children              |                 |                                                                                                             |
    |                       |                 |                                                                                                             |
    +-----------------------+-----------------+-------------------------------------------------------------------------------------------------------------+
+   | Quota warning         | integer         | Only available in :guilabel:`Advanced Mode`.                                                                |
```
Comment (wblock) [inline userguide/storage.rst]: Is this a duplicate of the one above?

File: userguide/storage.rst [hunk 9]
```diff
@@ -747,11 +747,31 @@ clicking the :guilabel:`Edit Options` button in
...
+   | alert at, %           |                 | Issues a warning alert when the dataset quota has reached the specifed value in percent.                    |
+   |                       |                 | To inherit the dataset values of the parent, leave blank. To disable this setting, enter *0*.               |
+   |                       |                 |                                                                                                             |
+   +-----------------------+-----------------+-------------------------------------------------------------------------------------------------------------+
+   | Quota critical        | integer         | Only available in :guilabel:`Advanced Mode`.                                                                |
```
Comment (wblock) [inline userguide/storage.rst]: Too many pastes?

File: userguide/storage.rst (hunk 8 above)
Comment (a-stjohn) [reply userguide/storage.rst]: No. In the old UI they appear how they are in the table.

File: userguide/storage.rst (hunk 9 above)
Comment (a-stjohn) [reply userguide/storage.rst]: No. In the old UI they appear how they are in the table.

File: userguide/intro.rst [hunk 10]
```diff
@@ -96,6 +96,11 @@ fixes.
 
 These major features are new in this version:
 
+* Change if an alert is shown when using a specific portion of space on
```
Comment (wblock) [inline userguide/intro.rst]: This is not so much a change as a new thing. So:
`` * Add options to show warning and critical alerts when a dataset reaches a certain percent of the allocated quota. New fields... ``

File: userguide/storage.rst [hunk 11]
```diff
@@ -711,90 +711,103 @@ clicking the :guilabel:`Edit Options` button in
...
-   |                       |                 |                                                                                                             |
-   +-----------------------+-----------------+-------------------------------------------------------------------------------------------------------------+
+   +-----------------------+-----------------+----------------+--------------------------------------------------------------------------------------------+
+   | Setting               | Value           | Advanced Mode  | Description                                                                                |
+   |                       |                 |                |                                                                                            |
+   +=======================+=================+================+============================================================================================+
+   | Dataset Name          | string          |                | Enter a mandatory unique name for the dataset.                                             |
```
Comment (wblock) [inline userguide/storage.rst]: Enter a unique name for the dataset. Required.

File: userguide/storage.rst [hunk 12]
```diff
@@ -711,90 +711,103 @@ clicking the :guilabel:`Edit Options` button in
...
+   |                       |                 |                |                                                                                            |
+   +-----------------------+-----------------+----------------+--------------------------------------------------------------------------------------------+
+   | Comments              | string          |                | Enter optional comments or notes about this dataset.                                       |
+   |                       |                 |                |                                                                                            |
+   +-----------------------+-----------------+----------------+--------------------------------------------------------------------------------------------+
+   | Sync                  | drop-down       |                | Sets the data write synchronization.                                                       |
```
Comment (wblock) [inline userguide/storage.rst]: s/Sets/Set/

**Pull request merged**